components/post_view/post_body/post_body.jsx

```diff
@@ -67,11 +62,6 @@ export default class PostBody extends React.PureComponent {
          */
         isEmbedVisible: PropTypes.bool,
 
-        /**
-         * Whether or not the post username can be overridden.
-         */
-        enablePostUsernameOverride: PropTypes.bool.isRequired,
```

Comment:
Here we can stop passing both `parentPostUser` and `enablePostUsernameOverride` from the index